pkg/router/template/plugin.go

```diff
@@ -236,7 +235,7 @@ func endpointsKeyFromParts(namespace, name string) string {
 func getPartsFromEndpointsKey(key string) (string, string) {
 	tokens := strings.SplitN(key, endpointsKeySeparator, 2)
 	if len(tokens) != 2 {
-		glog.Errorf("Expected separator %q not found in endpoints key %q", endpointsKeySeparator, key)
+		log.Error(nil, "expected separator not found in endpoints", "separator", endpointsKeySeparator, "key", key)
```

Comment:
Should probably keep the word "key" ("not found in endpoints key"), otherwise it is unclear where the separate was expected to be.